app/helpers/static_pages_helper.rb

```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
+end
```

Comment:
delete this file. because you dont use it

Reply:
Done